src/ui/actions/find-tests.ts

```diff
@@ -73,7 +71,6 @@ return [{
 // Mapper extracting the "name" and "message" properties from a thrown exception
 // for use in test failure messages.
 const JestExceptionMapper = `
-${Helpers}
```

Comment:
why is it okay to remove these? Seems unrelated

Reply:
Good catch - the `Helpers` string was:

```ts
export const Helpers = `
  const finalData = { frames: [], scopes: [], objects: [] };
  function addPauseData({ frames, scopes, objects }) {
    finalData.frames.push(...(frames || []));
    finalData.scopes.push(...(scopes || []));
    finalData.objects.push(...(objects || []));
  }
  function getTopFrame() {
    const { frame, data } = sendCommand("Pause.getTopFrame");
    addPauseData(data);
    return finalData.frames.find(f => f.frameId == frame);
  }
`;
```

Note that it adds `const finalData`.

I see a reference to `finalData` in one of the Jest test-finding mappers:

```ts
const JestExceptionMapper = `
${Helpers}
const { point, time, pauseId } = input;
const { frameId, location } = getTopFrame();
const { exception, data: exceptionData } = sendCommand("Pause.getExceptionValue");
addPauseData(exceptionData);
return [{
  key: point,
  value: { time, pauseId, location, exception, data: finalData },
}];
`;
```

So we should re-add `Helpers` somewhere, and restore those lines here.

Reply:
It's related. This is in `src/ui/actions` so it's Redux stuff, which this PR is deleting a bunch of. In this case, "Helpers" was referring to an export from `actions/logpoint` which was deleted in this PR :)

Reply:
Yeah, what I'm seeing is that the `JestExceptionMapper` string requires one of the variables created by that `Helper` string, so we need to bring it back.